code/modules/mob_spawn/corpses/job_corpses.dm

```diff
@@ -59,6 +59,11 @@
 	outfit = /datum/outfit/job/assistant
 	icon_state = "corpsegreytider"
 
+/obj/effect/mob_spawn/corpse/human/roboticist
+	name = "Roboticist"
```

Comment:
```suggestion
	name = JOB_ROBOTICIST
```

Reply:
While I am here, I'll do the same with the other jobs too